libmscore/layout.cpp

```diff
@@ -2658,6 +2660,9 @@ void Score::getNextMeasure(LayoutContext& lc)
             as.init(staff->keySigEvent(measure->tick()), staff->clef(measure->tick()));
 
             for (Segment& segment : measure->segments()) {
+                  // TODO? maybe we do need to process it here to make it possible to enable later
+                  //if (!segment.enabled())
```

Comment:
I mean these comments: what do they mean? Unfortunately I don't understand that even in the context of this PR so perhaps they could be elaborated somehow.

Reply:
Regarding the commented-out code I added, it's really not related.  I was looking at those loops while debugging, and saw that we weren't skipping disabled segments.  We've had other bugs that were fixed by making sure we did this, so I thought about just pre-emptively doing this here too.  But then I had second thoughts, like maybe something about this particular code actually does need to process those disable segments so that they are in whatever condition they need to be in to make it work to enable them later.  I didn't want to break something by adding code I didn't *know* we needed.  So I left the code in as a TODO, just to remind us that we should consider skipping disabled segments in general and some day, a bug might be submitted that will be fixed if we uncomment this.  To me, the comment is self-explanatory, but that's because I know the background :-)